src/components/dialogs/dialogUtils.js

```diff
@@ -71,6 +71,28 @@ export const useStyles = makeStyles((theme) => ({
     addDirectoryElements: {
         marginTop: -5,
     },
+    formChipItems1: {
```

Comment:
maybe more explicit names rather than xxx1 et xxx2 ?

Reply:
Removed (use of sx instead : see above)